java/daikon/Chicory.java

```diff
@@ -138,7 +144,7 @@
   /** flag to use if we want to turn on the static initialization checks **/
   public static final boolean checkStaticInit = true;
 
-  private static final boolean RemoteDebug = false;
+  private static /*final*/ boolean RemoteDebug = false;
```

Comment:
It's not your fault, but could you give this field a lowercase first letter to conform to the Java standard?
I would do it myself in master, but that would cause a conflict with your work.

Also, remove `final` rather than commenting it -- readers will be confused why the comment is there.